frontend/src/common/constansts/time.ts

```diff
@@ -1,4 +1,13 @@
 export const Time = {
     NOTIFICATION_DURATION: 4000,
     CURRENT_YEAR: new Date().getFullYear(),
-}
\ No newline at end of file
+}
+
+export const MONTHS = [
+    "January", "February", "March", "April", "May", "June",
+    "July", "August", "September", "October", "November", "December"
+];
+
+export const YEARS = Array.from({ length: 5 }, (_, i) => 2024 - i);
```

Comment:
Are you going to create PR each year to increase current year by one?

Few lines above you have Time. CURRENT_YEAR